app/controllers/application_controller.rb

```diff
@@ -29,12 +29,11 @@ class ApplicationController < ActionController::Base
   attr_reader :original_search_parameter
 
   def welcome
-    if (model_of_controller.first.nil? rescue false)
+    return if (model_of_controller.first.nil? rescue false)
```

Comment:
Should we get rid of this rescue?
I don't know why it is here, but if the tables are not migrated, it deserves a hard fail IMO.

Reply:
I'm guessing that's in case the `model_of_controller` throws and error or returns `nil` instead of a model

Reply:
probably could be changed to 
```suggestion
    return if model_of_controller&.any?
```
and perhaps internal exceptions should be handled by `model_of_controller` instead

Reply:
I've moved the rescue to the `model_of_controller` as `safe_constantize`, updated a commit message and the description.